### DISCUSSION TITLE
hx --tutor DOESN'T bring-up any tutor-file, in aarch64, built from cargo..

### DISCUSSION THREAD

**Discussion by libweirdness** [helix-editor/helix#11719] [Troubleshooting]
### Summary

I got Helix installed from source, & it works,
& hope to make its color-scheme less hard-to-read,

& the documentation for it says to run

hx  --tutor

( sorta like vim's tutor, I presume.. )

Which .. opens an empty text-file.

Blank.

Now I've used a browser FIND to save the text-version of the file,
& opened it up in Helix..

That should not be required of any newbie-to-Helix.

Please fix the git repo to include the tutor.txt file wherever it should be

( I've built mine in my user/src/ dir,
as it suggested )

& that Helix opens it,
when it's supposed to.

_ /\ _

### Reproduction Steps

hx --tutor

SHOULD open the tutor, ie the Helix editor with the tutor.txt file in it.

It opens hx, alright, but with nothing in it.

### Platform

Ubuntu server, 24.04LTS, aarch64

### Terminal Emulator

I'm not running X, so this is straight bash.

### Installation Method

source

### Helix Version

helix 24.7 (84fbadbd)

**Comment by the-mikedavis** [comment]
Did you configured the runtime directory while building from source? https://docs.helix-editor.com/master/building-from-source.html#configuring-helixs-runtime-files

**Reply by iampi31415** [reply]
Note that the docs include: `export HELIX_RUNTIME=~/src/helix/runtime`

But why `~/src` ? In my case, I had to modify it as:

`export HELIX_RUNTIME=~/helix/runtime`

I think they should simply say the path to wherever you cloned it.

https://docs.helix-editor.com/master/building-from-source.html#configuring-helixs-runtime-files
